tests/unit/updateTestResults.unitTest.ts

```diff
@@ -709,7 +703,6 @@ describe('updateTestResults', () => {
             const validationResponse =
               ValidationUtil.validateTestTypes(testToUpdate);
             expect(validationResponse).toBeDefined();
-            expect(validationResponse).not.toHaveLength(0);
```

Comment:
why is the test type check being removed?

Reply:
I've changed how errors are reported. The change to add the strip means that it won't be sending this error anymore. We don't think there are any downstream effects because of this but it is a risk.